openshift-ci/Dockerfile.tools

```diff
@@ -0,0 +1,37 @@
+FROM fedora:30 as build-tools
```

Comment:
I made some changes on the baremetal-deploy repo. One of those was using fedora:31.
Can you look at that?
Also, since it's shipping ginkgo, I'd remove the rule from the Makefile or (at least) remove the dependency of the `functests` rule from the `ginkgo` rule.